pkg/plugins/plugins.go

```diff
@@ -106,6 +118,86 @@ func (p PluginDTO) IncludedInSignature(file string) bool {
 	return true
 }
 
+func (p PluginDTO) Markdown(name string) []byte {
```

Comment:
Wouldn't it be nice if this called `p.File(...)` to get rid of some duplicated code?